test/message/promise_fast_handled_reject.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+
+const p1 = new Promise(function(res, rej) {
+  consol.log('One'); // eslint-disable-line no-undef
+});
+
+new Promise(function(res, rej) {
```

Comment:
I doubt it. If your JS vm is keeping it alive because of that I think you'll get memory leaks..

I've gone ahead and fixed it anyways.